### DISCUSSION TITLE
Issues

### DISCUSSION THREAD

**Discussion by sifunadonnex** [C-I-T-CLUB/CIT_CLUB_OFFICIAL_WEB_APP_-FRONTEND#6] [General]
Please feel free to comment and air your concern. Will be creating relative issues later today as no one has raised any concerns. Please be active so that we can move along.

**Comment by sifunadonnex** [comment]
Only 2 forks🤔, lets do so if you are to contribute

**Comment by kissingerbrian** [comment]
yes please
RoboticaVision
